donate/payments/views.py

```diff
@@ -438,7 +440,9 @@ def form_valid(self, form, send_data_to_basket=True):
                 ),
                 'payment_method_token': payment_method.token,
                 'price': form.cleaned_data['amount'],
-                'first_billing_date': now().date(),
+                'options': {
+                    "start_immediately": True
```

Comment:
```python
                    "start_immediately": True
```
Does this boolean ever end up in the Acoustic send mail function? I recall we had an issue with the serialization of Booleans before when sending a receipt email.

Reply:
Hi Ben! This boolean does not make its way to that function, this one is just for creating their subscription in Braintree